exampleSite/config.toml

```diff
@@ -53,7 +53,7 @@ disqusShortname = "yourdiscussshortname"
     custom_css = []
 
     # Custom JS
-    custom_js = []
+    custom_js = ["js/custom.js"]
```

Comment:
Is better to add this JS file into the <head> scope. Let's leave the custom_js field to user's scripts.

Reply:
Like CSS files, we should minify this JS file too.